### PR TITLE
Linguistic and style changes from user testing

### PR DESCRIPTION
**Pull request by farooqashar**
This PR does the following: 
- adds a new hero section to home page with some contextual app info
- changes linguistic wording on file upload and group vibe concepts
- adds a logo

### CODE REVIEW

File: frontend/src/views/HomeView.vue [hunk 1]
```diff
@@ -1,4 +1,25 @@
 <template>
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (farooqashar) [reply frontend/src/views/HomeView.vue]: yes, this should be fixed now; a result of some merge conflict things. thanks!

File: frontend/src/views/HomeView.vue [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+          <p class="mb-5">
+            QuickChat Is Your One-Stop Shop for Quick Messaging For Your Next
+            Team, Project, Lecture, and More.
+          </p>
+          <p>No Sign In. No Sign Up.</p>
+          <a href="#joinCreateChat">
+            <button class="btn btn-primary mt-5">Get Started</button>
+          </a>
+        </div>
+=======
```
[the remark this answers is not in the document]
Comment (farooqashar) [reply frontend/src/views/HomeView.vue]: yes, this should be fixed now; a result of some merge conflict things. thanks!

File: frontend/src/views/HomeView.vue [hunk 3]
```diff
@@ -45,22 +66,72 @@
             <input type="submit" class="btn btn-primary" value="Create" />
           </div>
         </form>
+>>>>>>> main
```
[the remark this answers is not in the document]
Comment (farooqashar) [reply frontend/src/views/HomeView.vue]: yes, this should be fixed now; a result of some merge conflict things. thanks!

**Review by hmerker — APPROVED** [review]
LGTM

**Pull request merged**